README.md

```diff
@@ -221,37 +235,49 @@ Swept-tuned detection is a standard spectrum analyzer measurement. The `SweptTun
 |`reference`|false|string|N/A|Data reference point, e.g., `"receiver input"`, `"antenna output"`, `"output of isotropic antenna"`.|
 
 ##### YFactorCalibration Object
-The `YFactorCalibration` object requires the following:  
+The purpose of the `YFactorCalibration` is to provide parameters and results for time-domain y-factor calibrations. The `YFactorCalibration` object contains the following name/value pairs:  
 
 |name|required|type|unit|description|
 |----|--------------|-------|-------|-----------|
-|`last_time_performed`|true|datetime|[ISO-8601](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#the-datetime-pair)|Date and time of last calibration.|
-|`calibrations`|false|array|dB|Receiver attenuations and corresponding gain and noise figure arrays equal in length to the [`sample_count`](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#annotation-segment-objects).|    
-|`reference`|false|string|N/A|Data reference point, e.g., `"receiver input"`, `"antenna output"`, `"output of isotropic antenna"`.|
+|`last_time_performed`|true|datetime|[ISO-8601](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#the-datetime-pair)|Date and time that calibration was performed.|
+|`frequencies`|false|array|Hz|Frequencies that y-factor calibrations are performed.|
+|`excess_noise_ratios`|false|array|dB|Excess noise ratio of calibrated noise source at `frequency` of y-factor calibration.|
```

Comment:
Somewhat ignorant here ... do the `frequencies` and `excess_noise_ratios` correspond to specific `calibrations`? If so, we need to establish that relationship though a multidimensional array, and should not be based on order.

Reply:
I don't understand the need for a multidimensional array to establish the relationship. The y-factor cal is performed at a number of `frequencies` . At each frequency, there is an excess noise ratio (enr) that is read off the calibrated noise source - the enr is needed to calculate the gain and noise figure at the frequency. enr can probably be excluded from metadata bc there is no reall need for it after gain and noise figure are calculated. It is important, however, to specify `frequencies`.